index.js

```diff
@@ -100,7 +101,7 @@ function entryPoint(staticHandler, file) {
 LiveServer.start = function(options) {
 	options = options || {};
 	var host = options.host || '0.0.0.0';
-	var port = options.port !== undefined ? options.port : 8080;
+	var port = options.port || 8080;
```

Comment:
This change would revert c592f94f2fafde84c566e3a28cc76074a0aa3f96, i.e. the explicit check for undefined is done in order to allow specifying port 0 for "don't care, give me random one". Should have added a comment though...